README.md

```diff
@@ -41,20 +41,18 @@ The following directories are going to be used as volumes (`docker-compose.yml`)
 - orjarinstall.py and sample orserver-add-l2pserver.zip is available under Assets https://github.com/ActianCorp/OpenROAD_docker/releases/tag/v0.0.1
 - orjarinstall_cfg.json file is available in `saveset` directory. This file will be copied to $II_SYTEM/ingres/files
 - The JSON file orjarinstall_cfg.json should contain a JSON object {...} with the following members (all optional):
-         "libu3gldir" - the directory shared libraries/DLLs are deployed into.
-             This directory should be contained in LD_LIBRARY_PATH (Linux) or PATH (Windows); default: $II_SYTEM/ingres/lib (Linux) or %II_SYSTEM%\ingres\bin (Windows)
-         "orjsonconfigdir" - the directory JSON config files for OpenROAD server applications are deployed into (default: $II_SYTEM/ingres/files/orjsonconfig)
-         "resourcedir" - the directory "resource*" directories (and their contents) are deployed into (default: $II_SYTEM/ingres)
-         "w4glappsdir" - the directory 4GL image files are deployed into; directory should be contained in II_W4GLAPPS_DIR (default: $II_SYTEM/ingres/w4glapps)
-- orserver-add-<appname>.zip is an archive containing the server application and other resources required
-  This archive can have the following subdirectories that will be processed:
-      - libu3gl - shared libraries/DLLs
-      - netutil - netutil command scripts
-      - orjsonconfig - JSON config files for OpenROAD server applications
-      - orserveradm_removeapp - JSON file to be used by the "orserveradm.py" script with RemoveApp command
-      - orserveradm_addapp - JSON file to be used by the "orserveradm.py" script with AddApp command
-      - resource* - Additional resource directories to be deployed
-      - w4glapps - OpenROAD Server application images
+	- "libu3gldir" : the directory shared libraries/DLLs are deployed into. This directory should be contained in LD_LIBRARY_PATH (Linux) or PATH (Windows); default: $II_SYTEM/ingres/lib (Linux) or %II_SYSTEM%\ingres\bin (Windows)
+	- "orjsonconfigdir" : the directory JSON config files for OpenROAD server applications are deployed into (default: $II_SYTEM/ingres/files/orjsonconfig)
+	- "resourcedir" : the directory "resource*" directories (and their contents) are deployed into (default: $II_SYTEM/ingres)
+	- "w4glappsdir" : the directory 4GL image files are deployed into; directory should be contained in II_W4GLAPPS_DIR (default: $II_SYTEM/ingres/w4glapps)
+- orserver-add-<appname>.zip is an archive containing the server application and other resources required for deployment. This archive can have the following subdirectories that will be processed:
```

Comment:
becuase this is markdown, and html can leak. I recommend replacing `<appname>` with `**appname**`, i.e. orserver-add-**appname**.zip

Reply:
Thanks, updated with **appname** as suggested.